packages/features/bookings/Booker/components/BookEventForm/CustomInputFields.tsx

```diff
@@ -0,0 +1,169 @@
+import { EventTypeCustomInputType } from "@prisma/client";
```

Comment:
Existing custom input fields logic, only abstracted it to it's own reusable file.